### ISSUE TITLE
Piano size

### ISSUE THREAD

**Issue by airdine** [onlaj/Piano-LED-Visualizer#532]
Does this project can be adapted to a smaller kid piano ? (like 30 keys)
Like using a smaller led strips and configuring start key and end key on piano ?

**Comment by dk865** [comment]
Yes, it's really easy to. Just make sure to get a smaller strip of lights.

**Comment by onlaj** [comment]
Yes, there are options to set number of leds, density, offsets etc.
